source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+* :doc:`denoise single-end <../plugins/available/dada2/denoise-single/>` accepts either single-end or unmerged paired-end data. If you give it unmerged paired-end data, it will only use the forward reads (and do nothing with the reverse reads).
+* :doc:`denoise-pyro <../plugins/available/dada2/denoise-pyro/>` accepts ion torrent data.
+
+Note that DADA2 may be slow on very large datasets. You can increase the number of threads to use with the ``--p-n-threads`` parameter.
+
+Denoising with deblur
+''''''''''''''''''''''
+
+The :doc:`deblur <../plugins/available/deblur/index>` plugin has two methods to denoise sequences:
+
+* :doc:`denoise-16S <../plugins/available/deblur/denoise-16S/>` denoises 16S sequences.
+* :doc:`denoise-other <../plugins/available/deblur/denoise-other/>` denoises other types of sequences.
+
+If you use ``denoise-16S``, deblur performs an initial positive filtering step where it discards any reads which do not have a minimum 60% identity similarity to sequences from the 85% OTU GreenGenes database.
+If you don't want to do this step, use the ``denoise-other`` method.
+
+deblur can currently only denoise single-end reads.
+It will accept unmerged paired-end reads as input, it just won't do anything with the reverse reads.
+Note that deblur *can* take in *merged* reads and treat them as single-end reads.
+
+OTU Clustering
+~~~~~~~~~~~~~~
+
+In this tutorial, we'll cover QIIME 2 methods that perform :doc:`de novo  <../plugins/available/vsearch/cluster-features-de-novo/>` and :doc:`closed reference <../plugins/available/vsearch/cluster-features-closed-reference/>` clustering.
+
+To cluster your sequences, you need to make sure that:
+
+-  paired-end reads are merged
+-  non-biological sequences are removed
+-  reads are all trimmed to the same length
+-  low-quality reads are discarded
+
+We discussed merging paired-end reads and removing non-biological sequences above (Sections `Merge reads`_ and `Remove non-biological sequences`_).
+
+Once your data is ready, you'll also need to dereplicate your reads before clustering.
+
+Length trimming
+'''''''''''''''
+
+If for some reason your raw reads are not already all the same length, you'll need to trim them to the same length before doing OTU clustering.
+There isn't currently a QIIME 2 function to trim reads to the same length without doing anything else, though you may be able to use functions from the ``cutadapt`` plugin to do something like that.
+(The reason for this is that the :ref:`QIIME 2 workflow <Denoising>` recommends first denoising reads - which involves a length trimming step - and then optionally passing the ASVs through a clustering algorithm.)
+
+Quality filtering
+'''''''''''''''''
+
+You can perform different types of quality filtering with the :doc:`quality filter <../plugins/available/quality-filter/index>` plugin.
+**TODO: what's the difference between the q-score and q-score-joined? Why are there two separate functions?**
```

Comment:
@wasade could you please answer this?

Reply:
One is for the single end semantic type, the other is for paired IIRC